drivers/SmartThings/zwave-switch/src/test/test_zooz_zen_30_dimmer_relay.lua

```diff
@@ -0,0 +1,723 @@
+local zw = require "st.zwave"
+local test = require "integration_test"
+local zw_test_utils = require "integration_test.zwave_test_utils"
+local test_utils = require "integration_test.utils"
+local capabilities = require "st.capabilities"
+local CentralScene = (require "st.zwave.CommandClass.CentralScene")({ version = 1 })
+local SwitchBinary = (require "st.zwave.CommandClass.SwitchBinary")({ version = 2 })
+local SwitchMultilevel = (require "st.zwave.CommandClass.SwitchMultilevel")({ version=4 })
+
+local zooz_zen_30_dimmer_relay_endpoints = {
+  {
+    command_classes = {
+      { value = zw.SWITCH_BINARY },
+      { value = zw.SWITCH_MULTILEVEL },
+      { value = zw.CENTRAL_SCENE }
+    },
+    command_classes = {
+      { value = zw.SWITCH_BINARY }
+    }
+  }
+}
+
+local mock_zooz_zen_30_dimmer_relay = test.mock_device.build_test_zwave_device({
+  profile = test_utils.get_profile_definition("zooz-zen-30-dimmer-relay.yml"),
+  zwave_endpoints = zooz_zen_30_dimmer_relay_endpoints,
+  zwave_manufacturer_id = 0x027A,
+  zwave_product_type = 0xA000,
+  zwave_product_id = 0xA008
+})
+
+local function test_init()
+  test.mock_device.add_test_device(mock_zooz_zen_30_dimmer_relay)
+end
+
+test.set_test_init_function(test_init)
+
+test.register_coroutine_test(
+  "Switch capability on command should evoke the correct Z-Wave SETs and GETs with dst_channel 0",
+  function()
+    test.timer.__create_and_queue_test_time_advance_timer(1, "oneshot")
+    test.socket.capability:__queue_receive({
+      mock_zooz_zen_30_dimmer_relay.id,
+      { capability = "switch", component = "main", command = "on", args = {} }
+    })
+
+    test.socket.zwave:__expect_send(
+      zw_test_utils.zwave_test_build_send_command(
+        mock_zooz_zen_30_dimmer_relay,
+        SwitchBinary:Set({
+          target_value = SwitchBinary.value.ON_ENABLE,
+          duration = 0
+        },
+        {
+          encap = zw.ENCAP.AUTO,
+          src_channel = 0,
+          dst_channels = {0}
+        })
+      )
+    )
+
+    test.wait_for_events()
+
+    test.wait_for_events()
```

Comment:
I guess You need only one line with `test.wait_for_events()`

Reply:
> I guess You need only one line with `test.wait_for_events()`

fixed